Formula/pastebinit.rb

```diff
@@ -25,6 +25,9 @@ def install
   end
 
   test do
-    system "date | pastebinit"
+    url = shell_output("echo \"Hello world at $(date)\" | pastebinit").chomp
+    assert_match %r{^http://pastebin\.com/}, url
+    # We can't actually fetch the URL to check the paste's success because pastebin
+    # blocks our fetches with curl, probably based on the user agent
```

Comment:
Yeah, but we don't want to defeat it. Our policy is to report homebrew/curl as the user agent, and respect the server side's handling of it.